cmd/minishift/cmd/start.go

```diff
@@ -506,6 +505,12 @@ func initStartFlags() *flag.FlagSet {
 		startFlagSet.String(configCmd.Nameserver.Name, "8.8.8.8", "Specify nameserver to use for the instance. Ignored if no IP address specified (experimental - Hyper-V only)")
 	}
 
+	if minishiftConfig.EnableExperimental {
+		startFlagSet.String(configCmd.ISOUrl.Name, minishiftConstants.B2dIsoAlias, "Location of the minishift ISO. Can be an URL, file URI or one of the following short names: [b2d centos minikube].")
+	} else {
+		startFlagSet.String(configCmd.ISOUrl.Name, minishiftConstants.B2dIsoAlias, "Location of the minishift ISO. Can be an URL, file URI or one of the following short names: [b2d centos].")
+	}
+
```

Comment:
@gbraad @praveenkumar @hferentschik do you think it adds any value? Honestly  I am not sure.

Reply:
For the time being... The actual problem is that the provsioner is not behind a toggle. Also the shorturl function is more effective in this regard.